### PR TITLE
[Coding Style] Enable rules Squiz.Functions.MultiLineFunctionDeclaration.*

### PR DESCRIPTION
**Pull request by sgiehl**
### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Comment by michalkleiner** [comment]
Thanks for the PR, @sgiehl. When we started these PSR12 PRs, we were doing one rule at a time to limit the number of files changed and to make the review easier. It seems like we went back on this a bit here. It's possible one rule here changes the majority of the files, but going over ~150 files makes the review unnecessarily bigger than it could be.
I guess it's ok to keep this PR since it's been already created, but could we do smaller changes for easier reviews going forward?

**Comment by sgiehl** [comment]
@michalkleiner All rules are around mutliline function declarations. For example the rule to only have one param per line would already change the majority of files. So not changing that in one PR would have ended in having multiple PRs with a similar amount of changed files. I thought it would be faster to have the review effort only once...

**Comment by michalkleiner** [comment]
Fair enough, if each line would change a similar large-ish amount of files, then it makes sense to review it only once. It could help to put that reasoning in the PR description 👍

File: plugins/ScheduledReports/tests/Integration/ApiTest.php [hunk 1]
```diff
@@ -112,8 +120,18 @@ public function getAttachments($report, $processedReports, $prettyDate) {
...
+            $report,
+            $contents,
+            $filename,
+            $prettyDate,
+            $reportSubject,
+            $reportTitle,
+            $additionalFiles,
+            $period,
+            $force
+) {
```
Comment (mneudert) [inline plugins/ScheduledReports/tests/Integration/ApiTest.php]:
```suggestion
        ) {
```

Interestingly zero spaces are accepted, but, if there is at least one space, the autofix goes to 8 space indentation :shrug:

**Review by mneudert — APPROVED** [review]

File: plugins/ScheduledReports/tests/Integration/ApiTest.php (hunk 1 above)
Comment (sgiehl) [reply plugins/ScheduledReports/tests/Integration/ApiTest.php]: Hm... Maybe that would be fixed by another rule at some point

**Pull request merged**